### PR TITLE
Feature/mizuki refactoring consensus

### PR DESCRIPTION
**Pull request by MizukiSonoko**
## What is this pull request?
A little fix & pass 
   
## Why do you implement it? Why do we need this pull request?
To pass CI
  
## How to use the features provided in the pull request?
Do you provide tests, or use case examples?
Describe how to use features in your pull request.

## Details/Features
List of features / major commits
- Remove `set_test`
- `using Transaction = protocol::Transaction`

### CODE REVIEW

File: irohad/torii/command_service.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 #include "torii/command_service.hpp"
```
Comment (motxx) [inline irohad/torii/command_service.cpp]: Fix `<torii/command_service.hpp>` ?

Comment (lebdron) [inline irohad/torii/command_service.cpp]: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
We should include it with `"` symbol

Comment (motxx) [inline irohad/torii/command_service.cpp]: If we'll use `"` symbol here, documents should be written at first, then all codes should follow the style (but currently, we use `<>` symbol for include path)

**Comment by lebdron** [comment]
Closing this PR, since code in `feature/consensus` was replaced by force push

**Pull request closed without merging**